normalization/test_ufonormalizer.py

```diff
@@ -1565,6 +1571,142 @@ def test_xmlConvertInt(self):
         self.assertEqual(xmlConvertInt(0o0000030), '24')
         self.assertEqual(xmlConvertInt(65536), '65536')
 
+    def test_duplicateUFO(self):
+        pass
+
+
+class SubpathTest(unittest.TestCase):
+    def __init__(self, methodName):
+        unittest.TestCase.__init__(self, methodName)
+        self.filename = 'tmp'
+        self.plistname = 'tmp.plist'
+
+    def setUp(self):
+        self.directory = tempfile.mkdtemp()
+        self.filepath = os.path.join(self.directory, self.filename)
+        self.plistpath = os.path.join(self.directory, self.plistname)
+
+    def tearDown(self):
+        shutil.rmtree(self.directory)
+
+    def createTestFile(self, text, num=None):
+        if num is None:
+            with open(self.filepath, 'w') as f:
+                f.write(text)
+        else:
+            for i in range(num):
+                filepath = self.filepath + str(i)
+                with open(filepath, 'w') as f:
+                    f.write(text)
+
+    def test_subpathJoin(self):
+        self.assertEqual(subpathJoin('a', 'b', 'c'),
+                         os.path.join('a', 'b', 'c'))
+        self.assertEqual(subpathJoin('a', os.path.join('b', 'c')),
+                         os.path.join('a', 'b', 'c'))
+
+    def test_subpathSplit(self):
+        self.assertEqual(subpathSplit(os.path.join('a', 'b')),
+                         os.path.split(os.path.join('a', 'b')))
+        self.assertEqual(subpathSplit(os.path.join('a', 'b', 'c')),
+                         os.path.split(os.path.join('a', 'b', 'c')))
+
+    def test_subpathExists(self):
+        self.createTestFile('')
+        self.assertTrue(subpathExists(self.directory, self.filepath))
+        self.assertFalse(subpathExists(self.directory, 'nofile.txt'))
+
+    def test_subpathReadFile(self):
+        text = 'foo bar™'
+        self.createTestFile(text)
+        self.assertEqual(text, subpathReadFile(self.directory, self.filename))
+
+    def test_subpathReadPlist(self):
+        data = dict([('a', 'foo'), ('b', 'bar'), ('c', '™')])
+        with open(self.plistpath, 'wb') as f:
+            f.write(dumps(data))
+        self.assertEqual(subpathReadPlist(self.directory, self.plistname),
+                         data)
+
+    def test_subpathWriteFile(self):
+        expected_text = 'foo bar™'
+        subpathWriteFile(expected_text, self.directory, self.filename)
+        with open(self.filepath, 'r') as f:
```

Comment:
if instead of trademark symbol here you had used a character which can't be encoded in cp1252, I would expect this test to fail on Windows.

Why are we not testing Windows with appveyor?